src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -91,9 +91,9 @@ export class CreateNewsComponent implements OnInit {
 
   public onSourceChange(): void {
     this.createNewsForm.get('source').valueChanges.subscribe(source => {
-      source.startsWith('http://') ||
-      source.startsWith('https://') ||
-      source.length === 0 ? this.isLink = false : this.isLink = true;
+      this.isLink = source.startsWith('http://') ||
+                    source.startsWith('https://') ||
+                    source.length === 0 ? false : true;
```

Comment:
```js
const baz = c ? false : true;
```
is actually `unneeded ternary` - read here [https://github.com/airbnb/javascript](https://github.com/airbnb/javascript)